Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contests/contest-solution-submit/ContestSolutionSubmitPage.tsx

```diff
@@ -262,7 +263,7 @@ const ContestSolutionSubmitPage = () => {
                 const formattedTime = calculatedTimeFormatted(moment.duration(remainingCompeteTime, 'millisecond'));
                 setRemainingTimeForCompete(formattedTime);
             } else {
-                setRemainingTimeForCompete(null);
+                setRemainingTimeForCompete(defaultRemainingTimeFormat);
```

Comment:
Can we just pass 0 duration to `calculatedTimeFormatted` and let the function do the rest?